laser_scan_matcher/CMakeLists.txt

```diff
@@ -20,8 +16,13 @@ set( ROS_CXX_DEPENDENCIES
 find_package(catkin REQUIRED COMPONENTS ${ROS_CXX_DEPENDENCIES})
 find_package(PCL REQUIRED QUIET)
 
+#Include csm project code and compile it
```

Comment:
I think we're no longer compiling the code upon the building of laser_scan_matcher package.